code/game/objects/items/RPD.dm

```diff
@@ -86,6 +86,7 @@ GLOBAL_LIST_INIT(RPD_recipes, list(
 		"Manual Valve"   = new /datum/pipe_info(PIPE_MVALVE, 			1, PIPE_BINARY),
 		"Digital Valve"  = new /datum/pipe_info(PIPE_DVALVE,			1, PIPE_BINARY),
 		"4-Way Manifold" = new /datum/pipe_info(PIPE_4WAYMANIFOLD,		1, PIPE_QUAD),
+		"Layer Manifold" = new /datum/pipe_info(PIPE_LAYER_MANIFOLD,	2, PIPE_UNARY),
```

Comment:
Indentation

Reply:
![image](https://user-images.githubusercontent.com/2003111/30891582-ee3f916c-a2e8-11e7-8368-f730e998c9e5.png)
not sure what's wrong